frameworks/g-cloud-12/questions/services/standardsCSASTARWhen.yml

```diff
@@ -13,6 +13,6 @@ type: text
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter the accreditation date.
   - name: under_character_limit
-    message: Your accreditation date must be no more than 50 characters.
```

Comment:
Query: should the entry of the date be in pre-set format, and thereby making a character limit redundant?

Reply:
We don't currently use date input types on the framework application questions 😿 I think it would be too complex to add at this stage in terms of backend validation, also it would prevent suppliers re-using their G11 answer when copying services.

Reply:
Saying that, I will add a tech debt card to make sure this gets fixed for future iterations.

Reply:
thanks.

Reply:
https://trello.com/c/Zf64qELt/1281-g-cloud-service-questions-asking-for-dates-are-using-text-input-not-date-input